app/sidekiq/publish_translation_job.rb

```diff
@@ -0,0 +1,13 @@
+class PublishTranslationJob
+  include Sidekiq::Job
+
+  def perform(id)
+    translation = Translation.find(id)
+    begin
+      translation.push_published_to_s3
+      translation.update!(is_published: true)
```

Comment:
I think marking `is_published: true` should be part of the `push_published_to_s3`